packages/tailwindcss-language-service/src/util/color.ts

```diff
@@ -275,7 +275,7 @@ export function getColorFromValue(value: unknown): culori.Color | KeywordColor |
   ) {
     return null
   }
-  const color = culori.parse(trimmedValue)
+  const color = tryParseColor(trimmedValue)
   return color ?? null
```

Comment:
I believe we can just do this immediately now that `tryParseColor` returns a `null` in case of an error.
```suggestion
  return tryParseColor(trimmedValue)
```